### ISSUE TITLE
training without pre-trained model

### ISSUE THREAD

**Issue by luhaijun** [BichenWuUCB/squeezeDet#87]
Thank you for the code. It's great.I want to train a result now, without using pre-trained model, If it can work, what should be changed or noted?If not, why not?
I am now training on Kitty data without the pre-trained model, but the accuracy rate is about 0.1%

**Comment by Timen** [comment]
The lower layers are frozen, which means they aren't being trained, only the top 2 layers are trained for detection. Thus if you don't initialise the lower layers with pretrained weights they are never trained and thus produce random results.

**Comment by muthiyanbhushan** [comment]
Hello,

Are you using the same dataset provided in readme.md file for training without pre-trained models?
Thanks.

**Comment by JUSTDODoDo** [comment]
hello,I have encountered the same problem as you. I deleted the input of the source code pre-training model, initialized with normal-distributed parameters, and trained hundreds of thousands of times to verify that there is still no precision.

**Comment by Timen** [comment]
Training an object detector from scratch is possible but not trivial, have a look at this paper: [DSOD: Learning Deeply Supervised Object Detectors from Scratch](https://arxiv.org/abs/1708.01241)

It is not possible to train squeezedet from scratch, you have to start with pretrained weights, I would also recommend looking at other more recent designs such as SSD Mobilenet, or a version of Retinanet.

**Comment by kemusa** [comment]
Just got to config/config.py and change `cfg.LOAD_PRETRAINED_MODEL` to `False`
